common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -193,6 +193,10 @@ def modulestore():
             settings.MODULESTORE['default'].get('OPTIONS', {})
         )
 
+        if settings.FEATURES.get('CUSTOM_COURSES_EDX'):
+            from ccx.modulestore import CCXModulestoreWrapper
```

Comment:
@cpennington: maybe, though I can't see how importing `ccx.modulestore` would cause `ccx.utils` to get imported too.  

At any rate, module scope import blows everything up, so I'm going to leave this where it is.  Do you want me to comment on it to ensure someone doesn't move it later?